README.md

```diff
@@ -154,6 +154,7 @@ This action can be configured to authenticate with GitHub App Installation or Pe
 | `HEAVILY_INVOLVED_CUTOFF` | False | 3 | Cutoff after which a mentor's comments in one issue are no longer counted against their total score |
 | `LABELS_TO_MEASURE`           | False    | `""`    | A comma separated list of labels to measure how much time the label is applied. If not provided, no labels durations will be measured. Not compatible with discussions at this time. |
 | `SEARCH_QUERY`                | True     | `""`    | The query by which you can filter issues/PRs which must contain a `repo:`, `org:`, `owner:`, or a `user:` entry. For discussions, include `type:discussions` in the query. |
+| `HIDE_ITEMS_CLOSED_COUNT`     | False    | False   | If set to `true`, the number of items closed metric will not be displayed in the generated Markdown file. |
```

Comment:
Does it make sense to put these in some sort of grouping/order?  I noticed that this got added to the bottom of the README, but is in the middle of the env example (with the other hides).

Reply:
Great catch, Thanks!! I've updated the order to match other areas (alphabetized).